net/modemmanager/files/modemmanager.proto

```diff
@@ -480,6 +481,25 @@ proto_modemmanager_setup() {
 		}
 	}
 
+	# set initial eps bearer settimgs (4G only)
+	[ -n "${gpp_epsbearer}" ] && [ "${gpp_epsbearer}" -eq 1 ] && {
+		echo "Setting inital EPS bearer with apn '${apn}'..."
+		# Append options to 'connectargs' variable
+		connectargs=""
+		append_param "apn=${apn}"
+		append_param "${iptype:+ip-type=${iptype}}"
+		append_param "${cliauth:+allowed-auth=${cliauth}}"
+		append_param "${username:+user=${username}}"
+		append_param "${password:+password=${password}}"
```

Comment:
We recently added a new "force" boolean setting in this operation. If set, the settings update is always done, regardless of whether the current settings are the same. Without this "force" option, MM will treat the request as a no-op if the settings are the same.